### ISSUE TITLE
`createStreams` returns a reset errorStream

### ISSUE THREAD

**Issue by liggitt** [kubernetes/kubernetes#13137]
There was a refactor that moved the creation of exec streams into its own function in `kubelet.go#createStreams`. That introduced a bug with this code:
```
			case api.StreamTypeError:
				errorStream = stream
				defer errorStream.Reset()
```

Previously, the deferred reset would occur after the stream was finished being used, but now that the stream creation occurs in its own function, it renders the stream useless.

cc @ncdc

**Comment by ncdc** [comment]
I am working on a fix for this

**Comment by ncdc** [comment]
This was fixed in #13322
